CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ PyO3 versions, please see the [migration guide](https://pyo3.rs/latest/migration
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [next] - Not yet released
+
+### Changed
+
+- `del` on `[setter]` defined class properties now raise `AttributeError`.
+
+### Fixed
+
+- Fix panic when calling `del` on a `[setter]` defined class property. [#1775](https://github.com/PyO3/pyo3/issues/1775)
```

Comment:
I think let's try to put these in a single changelog line, perhaps something like the below:

```suggestion

### Fixed

- Raise `AttributeError` to avoid panic when calling `del` on a `[setter]` defined class property. [#1775](https://github.com/PyO3/pyo3/issues/1775)
```